pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -43,6 +43,7 @@ type PodAutoscaler struct {
 	Spec PodAutoscalerSpec `json:"spec,omitempty"`
 
 	// Status communicates the observed state of the PodAutoscaler (from the controller).
+	// +kubebuilder:default={"serviceName": "", "metricsServiceName": "", "conditions": {{"type":"Active", "status": "Unknown", "reason": "Deploying",  "message": "", "lastTransitionTime": "1970-01-01T00:00:00Z"}, {"type":"Ready", "status": "Unknown", "reason": "Deploying", "message": "", "lastTransitionTime": "1970-01-01T00:00:00Z"}, {"type":"SKSReady", "status": "Unknown", "reason": "Deploying", "message": "", "lastTransitionTime": "1970-01-01T00:00:00Z"}, {"type":"ScaleTargetInitialized", "status": "Unknown", "reason": "Deploying", "message": "", "lastTransitionTime": "1970-01-01T00:00:00Z"}}}
 	// +optional
 	Status PodAutoscalerStatus `json:"status,omitempty"`
```

Comment:
Are we able to push this down into `PodAutoscalerStatus` ?

Reply:
@dprotaso you good with it like that? otherwise we would ned to find another way, webhook maybe?